components/org.wso2.carbon.privacy.forgetme.logs/src/main/java/org/wso2/carbon/privacy/forgetme/logs/instructions/LogFileInstruction.java

```diff
@@ -61,7 +61,7 @@ public ForgetMeResult execute(UserIdentifier userIdentifier, ProcessorConfig pro
         LogFileProcessor logFileProcessor = new LogFileProcessor();
 
         if (log.isDebugEnabled()) {
-            log.debug("File {} is being processed.", logFile.getName());
+            log.debug("File {} is being processed." + logFile.getName());
```

Comment:
Expectation seems to be printing a log line like `File somefile is being processed.`. But now will be printed as `File {} is being processed.somefile`

Reply:
Fixed with https://github.com/wso2/identity-anonymization-tool/pull/141